CONTRIBUTING.md

```diff
@@ -118,7 +150,102 @@ Each subdirectory contains topical subdirectories for each chapter in that secti
 Each topical subdirectory contains:
 
   - An *index.md* file that has the metadata discussed above plus the text for the section
-  - An *assets* subdirectory where graphical files for that topic are stored
+  - An *assets* subdirectory where graphical files for that topic are stored.
 No *assets* subdirectory is present if the topic has no graphics.
 
+## Submitting new content through a PR
+
+If you have forked and cloned the repository,
+you can modify the documentation or add new material
+by editing the markdown file(s) in the local clone of your fork
+and then submitting a PR.
+
+Note that you can also modify the source using the github editor.
+This is very useful when you want to fix a typo or make a small editorial change but, if you are doing significant writing, it is generally better to work on files in your local clone.
+
+The following sequence of steps is a reasonable workflow for creating new content on your local clone and pushing it to github to be reviewed and merged.
+
+1. Execute the following command from the root folder and leave it running in a shell:
+    ```
+    hugo server -D
+    ```
+2. Create a local branch for your changes.  Be sure to base your new branch on the contents of the `master` branch unless you intend to create a derivative PR:
+   ```
+   git branch master
```

Comment:
Some of these bits could be simplified via GitHub CLI: https://cli.github.com/
Maybe it is something to recommend for newcomer contributors in the next iteration of the guidelines